cse_hub/problems/evaluate.py

```diff
@@ -9,36 +9,36 @@ def check(ques, sol, file, time):
 	'''
 	# Just for refrence, prints the file being checked in terminal
 	print('Checking the file ', file, '_'*27)
-	
+
 	file = os.path.join(settings.BASE_DIR, 'problems', str(file))
 	command = 'python3'
 
 	# if given file is cpp file, we need to compile before running testcase
 	if str(file).endswith('.cpp'):
 		command = './a.out'
-		p = subprocess.Popen(f'g++ {file}', shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+		p = subprocess.Popen('g++ {file}', shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
 		for line in p.stderr.readlines():
-			# if we got compilation error, terminate 
+			# if we got compilation error, terminate
 			if line:
 				return 'CE'
 
 	# Just for refrence, prints the full command for file being checked to the terminal
-	print(f'evaluating using : cat {ques} | {command} {file}')
-	p = subprocess.Popen(f'cat {ques} | {command} {file}', shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-	
+	print('evaluating using : cat {ques} | {command} {file}')
+	p = subprocess.Popen('cat {ques} | {command} {file}', shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+
```

Comment:
- Please put f' ' back, it doesn't gives useful output on terminal, you might be getting errors because of using python2